tests/lava/lib/dnf/inputs/rate_code_spike_gen/test_process.py

```diff
@@ -46,8 +46,6 @@ def test_running(self) -> None:
         finally:
             spike_generator.stop()
```

Comment:
Are we just testing if everything runs without error here?

Reply:
Yes, it seems so. I removed this unit test as this was covered in another test.